### ISSUE TITLE
Error getting estimated metrics: unsupported operand type(s) for *: 'float' and 'NoneType'

### ISSUE THREAD

**Issue by IdoZach** [neuralmagic/sparsify#55]
Hi, after uploading and analyzing a onnx model with sparsify, it gives the following message.
The onnx model is correct (checked by the onnx checker) and can be used by other software. 

![image](https://user-images.githubusercontent.com/25578860/114450519-c7ad4b00-9bde-11eb-924d-5c89e372bc96.png)

**Comment by markurtz** [comment]
Thank you for filing this @IdoZach, a few questions from our side so we can look into reproducing and fixing:
- Could you provide a bit more info on if you ran the loss and performance analysis during setup? 
- What was the base framework the model was exported in and did you go through the ONNX export pathways in SparseML?
- What is the version of ONNX that is installed on your system and the ONNX opset the model was created with? 
- Finally, do you have a sample or similar model you might be able to share so we can reproduce using it?

**Comment by markurtz** [comment]
Hi @abuvaneswari, thank you for surfacing this issue again and supplying a sample model. We will look into this using the example over the next few days and get back to you with either a resolution or an estimate on time to resolution.

**Comment by markurtz** [comment]
@abuvaneswari and @IdoZach this should be fixed on the main branch now. Nightlies will be created tonight and you will be able to test with `pip install sparsify-nightly`. Additionally, the 0.4 release will be going out in the next few days with this change on it. Let us know if you have any further issues!

Thanks,
Mark

**Comment by abuvaneswari** [comment]
@markurtz , thank you. I did the pip install sparsify-nightly. 

However, when I access the sparsify interface (thru its webserver interface) and upload an onnx model, I get the following error at the 3rd step of 'profile model':
The deepsparse package must be installed for CPU performance profiling

I have deepsparse installed already... I tried v0.3.1 as well as the nightly versions of deepsparse. but for some reason, sparsify is not detecting these packages to be installed in the same env and I get the error in both cases.
